CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/importing/file/GraphMLImportProcessor.java

```diff
@@ -282,12 +284,20 @@ public void process(final PluginParameters parameters, final File input, final R
         if(!processingErrors.isEmpty()){
             
             // Count distinct processing errors
-            Map<String, Long> processingErrorTypes = processingErrors.stream()
+            final Map<String, Long> processingErrorTypes = processingErrors.stream()
                 .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()));
 
-            String errorMsg = processingErrorTypes.entrySet().stream()
-                .map(e -> e.getValue() + " " + e.getKey())
-                .collect(Collectors.joining("\n"));
+            final String errorMsg = processingErrorTypes.entrySet().stream()
+                    .map(e -> {
+                        String plural = " doesn't ";
```

Comment:
I think the space at the end here can be removed since the string it is being inserted into has a space afterwards